main.go

```diff
@@ -46,6 +54,10 @@ func init() {
 func main() {
 	flag.Parse()
 
+	if platform != "ubuntu" && platform != "windows" {
```

Comment:
See my earlier questions with regards to this. If, you would like to keep it the way it is, would you mind making "ubuntu" an exported constant inside the `bosh` (or `monit`) package?  Also, the string "windows" is not being used anywhere. So, why do you need this check at all?